Sources/UIAlertControllerView.swift

```diff
@@ -97,9 +163,18 @@ class UIAlertControllerButton: Button {
         }
     }
 
+    var titleLabelOriginX: CGFloat?
```

Comment:
can you add a code comment describing why we need this?

Reply:
Thanks for adding the comment! Currently it seems to be a "what" comment rather than "why" though – I still don't understand why this is necessary. Could you write the comment in terms of "what would happen if we didn't have this"?